integration/system/options_test.go

```diff
@@ -0,0 +1,51 @@
+package system
+
+import (
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/internal/test/daemon"
+
+	req "github.com/docker/docker/internal/test/request"
+	"gotest.tools/assert"
+)
+
+func TestAPIOptionsRoute(t *testing.T) {
+	defer setupTest(t)()
+
+	resp, _, err := req.Do("/", req.Method(http.MethodOptions))
+	assert.NilError(t, err)
+	assert.Equal(t, resp.StatusCode, http.StatusOK)
+}
+
+func TestAPIGetEnabledCORS(t *testing.T) {
+	defer setupTest(t)()
+
+	d := daemon.New(t)
+
+	d.Start(t, "--api-cors-header='*'")
```

Comment:
The problem here is that this works when executed in a shell (where the shell would strip the single quotes (`'`)  (but also expand `*` to files in the same directory), but the integration test is not executed in a shell; because of that, the quotes are not stripped.

However, because there's no shell, there's also no need to use quotes (because `*` are not expanded / glob'ed), so this will work:

```suggestion
	d.Start(t, "--api-cors-header=*")
```

Or, this:

```suggestion
	d.Start(t, "--api-cors-header, "*")
```

Double checked; with that change (and the updated `assert`); the test still works;

```
make TEST_FILTER=TestAPIGetEnabledCORS test-integration

....


=== RUN   TestAPIGetEnabledCORS
--- PASS: TestAPIGetEnabledCORS (0.75s)
    options_test.go:28: Creating a new daemon at: /go/src/github.com/docker/docker/bundles/test-integration/TestAPIGetEnabledCORS
    daemon.go:336: [ddd936e8e062d] waiting for daemon to start
    daemon.go:336: [ddd936e8e062d] waiting for daemon to start
    daemon.go:364: [ddd936e8e062d] daemon started
    daemon.go:472: [ddd936e8e062d] Stopping daemon
    daemon.go:307: [ddd936e8e062d] exiting daemon
    daemon.go:459: [ddd936e8e062d] Daemon stopped
PASS
```